src/psyclone/tests/domain/lfric/lfric_builtins_test.py

```diff
@@ -2881,6 +2892,26 @@ def test_int_X(tmpdir, monkeypatch, annexed, dist_mem):
         assert output_dm_2 in code
 
 
+def test_int_X_precision(monkeypatch):
+    '''Test that the builtin picks up and creates correct code for a
+    scalar with precision that is not the default i.e. not i_def. At
+    the moment there is no other integer precision so we make one
+    up. However, this does mean that we can't check whether it
+    compiles.
+
+    '''
+    _, invoke_info = parse(os.path.join(BASE_PATH,
+                                        "15.10.3_int_X_builtin.f90"),
+                           api=API)
+    psy = PSyFactory(API).create(invoke_info)
+    # Test string method
+    first_invoke = psy.invokes.invoke_list[0]
+    kern = first_invoke.schedule.children[0].loop_body[0]
+    monkeypatch.setattr(kern.args[0], "_precision", "i_solver")
+    code = str(psy.gen)
+    assert ("USE constants_mod, ONLY: i_solver, i_def" in code)
```

Comment:
pylint complains that the parentheses are unnecessary on this line and the next.

Reply:
Oops.